src/reset-password/controller/redirect.ts

```diff
@@ -1,8 +1,38 @@
 import Controller from '@curveball/controller';
+import { Context } from '@curveball/core';
+import log from '../../log/service';
+import { EventType } from '../../log/types';
+import * as UserService from '../../user/service';
+import { User } from '../../user/types';
+import { resetPasswordForm } from '../formats/redirect';
 
-class ChangePasswordController extends Controller {
+class ResetPasswordController extends Controller {
 
+  async get(ctx: Context) {
 
```

Comment:
Both get and post should check if there is something inside ctx.state.session.resetPasswordUser and throw an error if there isnt